bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerMappings.kt

```diff
@@ -55,7 +55,7 @@ class BusinessPartnerMappings {
         return BusinessPartnerOutputDto(
             externalId = entity.sharingState.externalId,
             nameParts = entity.nameParts,
-            identifiers = entity.identifiers.map(::toIdentifierDto),
+            identifiers = entity.identifiers.filter { it.businessPartnerType == BusinessPartnerType.GENERIC }.map(::toIdentifierDto),
```

Comment:
Identifiers could be filtered by determined address type:

- Legal Address: Only legal entity identifiers
- Site: No identifiers:
- Additional Address: Only address identifiers

Reply:
Done.